tools/packaging/kernel/build-kernel.sh

```diff
@@ -127,7 +127,12 @@ get_tee_kernel() {
 	mkdir -p ${kernel_path}
 
 	[ -z "${kernel_url}" ] && kernel_url=$(get_from_kata_deps "assets.kernel.${tee}.url")
-	kernel_tarball="${version}.tar.gz"
+
```

Comment:
My suggestion is to assume `kernel_tarball="linux-${version}.tar.gz"` is expected, otherwise add the 'tarball' property in the `versions.yml`. Something like:

```
kernel_tarball="linux-${version}.tar.gz"
tarball_name=$(get_from_kata_deps "assets.kernel.${tee}.tarball")
[ -z "$tarball_name" ] || kernel_tarball="$tarball_name"
```
Then change the TDX properties in `versions.yaml`:
```
diff --git a/versions.yaml b/versions.yaml
index eb6753293..b76f6e104 100644
--- a/versions.yaml
+++ b/versions.yaml
@@ -158,6 +158,7 @@ assets:
       description: "Linux kernel that supports TDX"
       url: "https://github.com/intel/linux-kernel-dcp/archive/refs/tags"
       tag: "SPR-BKC-PC-v9.6"
+      tarball: "SPR-BKC-PC-v9.6.tar.gz"
     sev:
       description: "Linux kernel with efi_secret support"
       url: "https://github.com/confidential-containers-demo/\
```

Reply:
done